### DISCUSSION TITLE
How to run docker-compose test and why it fails?

### DISCUSSION THREAD

**Discussion by Jeevananthan-23** [dotnet-architecture/eShopOnContainers#2014] [Q&A]
Hi, I tried to run docker-compose with unit test locally it failed and due to connection, I lost any idea why it's happening.
https://github.com/dotnet-architecture/eShopOnContainers/blob/4596d7aa99f841206e486e910d20b525cd48daf2/src/docker-compose-tests.yml#L7

**Comment by behdad088** [comment]
Hi @Jeevananthan-23, What is the error message you get?
